client/network/src/lib.rs

```diff
@@ -454,3 +454,106 @@ async fn wait_to_announce<Block: BlockT>(
 		);
 	}
 }
+
+/// Creates a new background task to wait for the relay chain to sync up and retrieve the parachain header
+pub async fn warp_sync_get<B, RCInterface>(
```

Comment:
Both of these functions can be moved to the `cumulus-service`. There is no need that they are here. Otherwise this function should be renamed and get some better docs.